Resources/Prototypes/Catalog/Fills/Lockers/misc.yml

```diff
@@ -141,6 +141,7 @@
     - id: CyberPen
     - id: CigPackSyndicate
     - id: ClothingBackpackDuffelSyndicatePyjamaBundle
+    - id: ClothingBackpackDuffelSyndicateCatMaidBundle
```

Comment:
```suggestion
```

The joke is not funny if it's not rare. Just remove it from the maints loot completely.